src/common/cloudinary.ts

```diff
@@ -53,3 +53,35 @@ const avatarPreset = 'avatar';
 
 export const uploadAvatar = (userId: string, stream: Readable) =>
   uploadFile(`${avatarPreset}_${userId}`, avatarPreset, stream);
+
+const postPreset = 'post_image';
+
+export const uploadPostFile = (
+  name: string,
+  stream: Readable,
+): Promise<string> =>
+  new Promise((resolve, reject) => {
+    const outStream = cloudinary.v2.uploader.upload_stream(
+      {
+        public_id: name,
+        upload_preset: postPreset,
+      },
+      (err, callResult) => {
+        if (err) {
+          return reject(err);
+        }
+
+        return resolve(
+          cloudinary.v2.url(callResult.public_id, {
```

Comment:
Can you also check if you can provide `sign_url: true` in the params or something like that. I'd like to start using signed urls when possible